packages/pipeline-editor/src/SubmitNotebookButtonExtension.tsx

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2018-2020 IBM Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { NotebookParser } from '@elyra/application';
+import { showFormDialog } from '@elyra/ui-components';
+import { JupyterFrontEnd } from '@jupyterlab/application';
+import { Dialog, ToolbarButton } from '@jupyterlab/apputils';
+import { DocumentRegistry } from '@jupyterlab/docregistry';
+import { INotebookModel, NotebookPanel } from '@jupyterlab/notebook';
+
+import { IDisposable } from '@lumino/disposable';
+import * as React from 'react';
+
+import { dialogWidget } from './dialogWidget';
+import NotebookSubmissionDialog from './NotebookSubmissionDialog';
+import { PipelineService } from './PipelineService';
+import Utils from './utils';
+
+/**
+ * Submit notebook button extension
+ *  - Attach button to notebook toolbar and launch a dialog requesting
+ *  information about the remote location to where submit the notebook
+ *  for execution
+ */
+export class SubmitNotebookButtonExtension
+  implements DocumentRegistry.IWidgetExtension<NotebookPanel, INotebookModel> {
+  private panel: NotebookPanel;
+
+  constructor(app: JupyterFrontEnd) {
+    this.app = app;
```

Comment:
I don't think this is used any more (probably remnant code), so we can probably remove this

Reply:
I removed this already in a follow-up PR I'm working on

Reply:
It's probably fine to remove it here as well, shouldn't break any diffs in your followup PR (I've done it before)